src/react/components/common/keyboardBinding/keyboardBinding.tsx

```diff
@@ -13,10 +13,15 @@ export class KeyboardBinding extends React.Component<IKeyboardBindingProps> {
     private deregisterBinding: () => void;
 
     public componentDidMount() {
-        this.deregisterBinding = this.context.keyboard.addHandler(
-            this.props.keyEventType || KeyEventType.KeyDown,
-            this.props.accelerator,
-            this.props.onKeyEvent);
+        if (this.context && this.context.keyboard) {
+            this.deregisterBinding = this.context.keyboard.addHandler(
+                this.props.keyEventType || KeyEventType.KeyDown,
+                this.props.accelerator,
+                this.props.onKeyEvent,
+            );
+        } else {
+            console.warn("Keyboard Mananger context cannot be found - Keyboard binding has NOT been set.");
```

Comment:
Localization

Reply:
This is more of a developer warning and warns the developer if they attempt to use the `KeyboardBinding` outside of a `KeyboardManager` component.